### PR TITLE
Dialog tweaks & multi-line copyright input (#23325)

### PR DESCRIPTION
**Pull request by mathesoncalum**
Resolves (partially): #23325 

Copyright & page number style settings coming up shortly...

### CODE REVIEW

File: src/project/view/projectpropertiesmodel.h [hunk 1]
```diff
@@ -65,12 +65,14 @@ class ProjectPropertiesModel : public QAbstractListModel
     enum Roles {
         PropertyName = Qt::UserRole + 1,
         PropertyValue,
-        IsStandardProperty
+        IsStandardProperty,
+        UseTextArea
```
Comment (Eism) [inline src/project/view/projectpropertiesmodel.h:69]: I think `IsMultiLineEdit` is a better name

File: src/project/qml/MuseScore/Project/internal/NewScore/GeneralInfoView.qml [hunk 2]
```diff
@@ -125,19 +126,37 @@ Column {
             navigation.panel: root.navigationPanel
             navigation.column: 3
         }
+    }
 
-        GeneralInfoItem {
-            id: copyrightInfo
+    Column {
```
Comment (Eism) [inline src/project/qml/MuseScore/Project/internal/NewScore/GeneralInfoView.qml:131]: It's probably better to move this into a separate file

File: src/project/qml/MuseScore/Project/internal/NewScore/GeneralInfoView.qml [hunk 3]
```diff
@@ -30,11 +30,12 @@ import MuseScore.Project 1.0
 Column {
     id: root
 
-    property alias title: titleInfo.info
-    property alias subtitle: subtitleInfo.info
-    property alias composer: composerInfo.info
-    property alias lyricist: lyricistInfo.info
-    property alias copyright: copyrightInfo.info
+    // Use hints by default, update using onInfoChanged
```
Comment (Eism) [inline src/project/qml/MuseScore/Project/internal/NewScore/GeneralInfoView.qml:33]: Looks strange :)
Maybe 
`property string title: Boolean(titleInfo.info) ? titleInfo.info : titleInfo.hint`

Comment (mathesoncalum) [reply src/project/qml/MuseScore/Project/internal/NewScore/GeneralInfoView.qml:33]: Sure - done!

**Review by Eism — APPROVED** [review]

**Comment by zacjansheski** [comment]
Tested on MacOS 14, Windows 11, Ubuntu 22.04.3. Approved

Just in case there is an accessibility concern, when going through project properties with arrow keys, selection gets stuck on copyright and you need to Esc to proceed.

https://github.com/musescore/MuseScore/assets/69917893/127edda2-c158-4475-8ac6-32a8002c48ea

**Review by zacjansheski — APPROVED** [review]

**Pull request merged**